### ISSUE TITLE
Not working with Scala 2.11.x?

### ISSUE THREAD

**Issue by gvolpe** [codacy/sbt-codacy-coverage#7]
I'm having problems trying to use the codacy plugin. I'm using Scala 2.11.7, OracleJDK 8 and SBT 0.13.8. This is the stack trace when I run 'sbt clean coverage test':

warn]   ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: com.typesafe.play#play-ws_2.10;2.3.8: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn]  Note: Unresolved dependencies path:
[warn]      com.typesafe.play:play-ws_2.10:2.3.8
[warn]        +- com.codacy:codacy-api-scala_2.10:1.0.0
[warn]        +- com.codacy:coverage-parser_2.10:1.0.0
[warn]        +- com.codacy:sbt-codacy-coverage:1.1.0 (scalaVersion=2.10, sbtVersion=0.13) (/home/gvolpe/development/workspace/light-play-rest-api/project/plugins.sbt#L9-10)
[warn]        +- default:light-play-rest-api-build:0.1-SNAPSHOT (scalaVersion=2.10, sbtVersion=0.13)
sbt.ResolveException: unresolved dependency: com.typesafe.play#play-ws_2.10;2.3.8: not found

I'm working on this project https://github.com/gvolpe/light-play-rest-api

**Comment by mrfyda** [comment]
Hello @gvolpe 

Did you include the Typesafe repository resolver?

```
resolvers += "Typesafe Repository" at "https://repo.typesafe.com/typesafe/releases/"
```

**Comment by gvolpe** [comment]
Hi @mrfyda 

I forgot it. It's working now, but it's downloading many dependencies of play_2.10. Why do you think is this happening? This is the log:

15:51:42light-play-rest-api:master sbt clean coverage test
[info] Loading project definition from /home/gvolpe/development/workspace/light-play-rest-api/project
[info] Updating {file:/home/gvolpe/development/workspace/light-play-rest-api/project/}light-play-rest-api-build...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] downloading https://repo.typesafe.com/typesafe/releases/com/typesafe/play/play-ws_2.10/2.3.8/play-ws_2.10-2.3.8.jar ...
[info]  [SUCCESSFUL ] com.typesafe.play#play-ws_2.10;2.3.8!play-ws_2.10.jar (10100ms)
[info] downloading https://repo.typesafe.com/typesafe/releases/com/typesafe/play/play_2.10/2.3.8/play_2.10-2.3.8.jar ...
[info]  [SUCCESSFUL ] com.typesafe.play#play_2.10;2.3.8!play_2.10.jar (27878ms)
[info] downloading https://repo.typesafe.com/typesafe/releases/com/typesafe/play/play-iteratees_2.10/2.3.8/play-iteratees_2.10-2.3.8.jar ...
[info]  [SUCCESSFUL ] com.typesafe.play#play-iteratees_2.10;2.3.8!play-iteratees_2.10.jar (10398ms)
[info] downloading https://repo.typesafe.com/typesafe/releases/com/typesafe/play/play-json_2.10/2.3.8/play-json_2.10-2.3.8.jar ...
[info]  [SUCCESSFUL ] com.typesafe.play#play-json_2.10;2.3.8!play-json_2.10.jar (9409ms)
[info] downloading https://repo1.maven.org/maven2/com/typesafe/akka/akka-slf4j_2.10/2.3.4/akka-slf4j_2.10-2.3.4.jar ...
[info]  [SUCCESSFUL ] com.typesafe.akka#akka-slf4j_2.10;2.3.4!akka-slf4j_2.10.jar (599ms)
[info] downloading https://repo1.maven.org/maven2/org/scala-stm/scala-stm_2.10/0.7/scala-stm_2.10-0.7.jar ...
[info]  [SUCCESSFUL ] org.scala-stm#scala-stm_2.10;0.7!scala-stm_2.10.jar (3329ms)
[info] downloading https://repo.typesafe.com/typesafe/releases/com/typesafe/play/play-functional_2.10/2.3.8/play-functional_2.10-2.3.8.jar ...
[info]  [SUCCESSFUL ] com.typesafe.play#play-functional_2.10;2.3.8!play-functional_2.10.jar (7661ms)
[info] downloading https://repo.typesafe.com/typesafe/releases/com/typesafe/play/play-datacommons_2.10/2.3.8/play-datacommons_2.10-2.3.8.jar ...
[info]  [SUCCESSFUL ] com.typesafe.play#play-datacommons_2.10;2.3.8!play-datacommons_2.10.jar (5274ms)
[info] Done updating.
[info] Set current project to light-play-rest-api (in build file:/home/gvolpe/development/workspace/light-play-rest-api/)
[success] Total time: 0 s, completed 29-Jul-2015 15:55:20
[success] Total time: 0 s, completed 29-Jul-2015 15:55:20

I'm using Play 2.4.2 and Scala 2.11.7.

Thanks,
Gabriel.

**Comment by mrfyda** [comment]
Yes @gvolpe.
The sbt-codacy-coverage plugin is only compiled and published for scalaVersion 2.10.4
As a result it'll download all the play 2.10 dependencies.

**Comment by gvolpe** [comment]
Ok, thank you @mrfyda
